<launch>
  <node pkg="rosgui" type="rosgui" name="rosgui" output="screen">
    <param name="robotPoseTopicName" type="string" value="/odometry/filtered_map" />
    <param name="globalPlanTopicName" type="string" value="/TrajectoryPlannerROS/global_plan" />
    <param name="originX" value="-10.0" />
    <param name="originY" value="-10.0" />
    <param name="originZ" value="0.0" />
  </node>
</launch>
